farm/modeling/language_model.py

```diff
@@ -354,7 +354,10 @@ def load(cls, pretrained_model_name_or_path, language=None, **kwargs):
 
         """
         albert = cls()
-        albert.name = pretrained_model_name_or_path
+        if "farm_lm_name" in kwargs:
+            albert.name = kwargs["farm_lm_name"]
+        else:
+            albert.name = pretrained_model_name_or_path
```

Comment:
I think its ok for now, but having unrelated changes in a PR is not best practice.